Sources/PointFree/Routes.swift

```diff
@@ -55,6 +56,38 @@ private func route<I, A, Route>(
     }
 }
 
+private let relatedDomains = [
+  "pointfree.co",
```

Comment:
What if we did the reverse and redirect anything that doesn't match to www.pointfree.co?

``` swift
private let hosts = [
  "www.pointfree.co",
  "localhost",
  "0.0.0.0",
  "127.0.0.1",
]
```

Reply:
Good idea! Will do